### DISCUSSION TITLE
Gltf αρχείο, Deliverable 3

### DISCUSSION THREAD

**Discussion by ThanosSl** [merkourisa/Augmented-Reality#25] [Q&A]
Θέλω να ανεβάσω το gltf αρχείο, πύργο του Eiffel και μου λέει ότι είναι πολύ μεγάλο, τι μπορώ να κάνω;

**Comment by merkourisa** [comment]
Καλησπέρα, οι ερωτήσεις που αφορούν στο 3ο παραδοτέο πρέπει να γίνονται στο αντίστοιχο https://github.com/merkourisa/Augmented-Reality/issues/3.

Χρησιμοποίησε το github desktop για να ανεβάσεις το αρχείο
